setup.py

```diff
@@ -43,13 +43,14 @@
     install_requires = [
         'semver>=2.7.0,<3.0',
         'jsonschema>=2.6.0,<3.0',
-        'jinja2>=2.8.1,<3.0',
+        'jinja2>=2.8.1',
         'pyyaml>=4.2b1',
         'marshmallow>=2.13.6,<3.0',
         'networkx>=2.4,<2.5',
         'xmltodict>=0.11.0,<1.0',
         'six>=1.11.0,<2.0',
         'markupsafe==2.0.1',
+        'importlib-metadata<5.0',
```

Comment:
What was this needed for?

Reply:
Test failures in newer versions of importlib-metadata. I can perform this release tmrw morning.